output/all.go

```diff
@@ -5,11 +5,15 @@ import (
 )
 
 // All is the list of all baker outputs.
-var All = []baker.OutputDesc{
-	DynamoDBDesc,
-	FileWriterDesc,
-	NopDesc,
-	OpLogDesc,
-	StatsDesc,
-	WebSocketDesc,
+var All = []baker.OutputDesc{}
+
+func init() {
+	All = append(All, []baker.OutputDesc{
+		DynamoDBDesc,
+		FileWriterDesc,
+		NopDesc,
+		OpLogDesc,
+		StatsDesc,
+		WebSocketDesc,
+	}...)
```

Comment:
I don't think this should be changed, first because it's always preferable to not use `init` if possible, second because the result is the same as the previous code

Reply:
ok